### PR TITLE
BUG: Fix ctk::vtkImageDataToQImage function

### PR DESCRIPTION
**Pull request by lassoan**
Bytes per line must be specified to make sure the image is always converted correctly.

The problem was reported and solution was proposed here: https://discourse.slicer.org/t/ctk-vtkimagedatatoqimage-generates-weird-results/25881/2

### CODE REVIEW

File: Libs/Visualization/VTK/Widgets/ctkVTKWidgetsUtils.cpp [hunk 1]
```diff
@@ -99,33 +99,37 @@ QImage ctk::vtkImageDataToQImage(vtkImageData* imageData)
...
+    }
+
+  int bytesPerLine = imageData->GetIncrements()[1]; // line increment
 
-  unsigned char* qtImageBuffer = image.bits();
-  memcpy( qtImageBuffer,
-    imageData->GetPointData()->GetScalars()->GetVoidPointer(0),
-    numberOfScalarComponents * width * height);
+  // Create an image from a buffer. The QImage object does not take ownership of the voxel buffer.
+  QImage image(static_cast<uint8_t*>(imageData->GetPointData()->GetScalars()->GetVoidPointer(0)),
```
Comment (finetjul) [inline Libs/Visualization/VTK/Widgets/ctkVTKWidgetsUtils.cpp:127]: shouldn't it be a `reinterpret_cast` rather than a `static_cast` ?

Comment (lassoan) [reply Libs/Visualization/VTK/Widgets/ctkVTKWidgetsUtils.cpp]: fixed!

**Comment by finetjul** [comment]
LGTM, thanks !

**Comment by jcfr** [comment]
Thanks @finetjul for the review :pray:

**Pull request merged**